universal-login-relayer/src/integration/sql/migrations/20200324000001_alter_deployments.js

```diff
@@ -0,0 +1,17 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
+  await knex.schema.alterTable('deployments', (table) => {
+    table.timestamp('created_at').notNullable().defaultTo('2015-07-30');
```

Comment:
I think default should have been setted knex.now, and if we want '2015-07-30' for existing rows we should update created_at value in all existing rows to '2015-07-30' after creating column

Reply:
Agree